backend/benefit/applications/api/v1/serializers/application.py

```diff
@@ -511,7 +511,7 @@ def _get_status_change_timestamp(self, obj, to_status=None):
         else:
             return None
 
-    ADDITIONAL_INFORMATION_DEADLINE = timedelta(days=14)
+    ADDITIONAL_INFORMATION_DEADLINE = timedelta(days=7)
```

Comment:
One thought I had about this simple fix: since the date always comes from the serializer method on the fly, when this feature goes live it automatically shortens the deadline for previously sent applications already in the database in that status as well. If that's a problem, some transitional period logic is needed.